pyrobuf/parse_proto3.py

```diff
@@ -0,0 +1,51 @@
+from parse_proto import *
```

Comment:
Agreed. I am down on relative imports these days - setting `PYTHON_PATH` and using `pyrobuf.parse_proto` seems like the best solution.